lich.rbw

```diff
@@ -2397,8 +2411,11 @@ module Games
                   rescue
                     unless $!.to_s =~ /invalid byte sequence/
                       # Fixes invalid XML with nested single quotes in it such as:
+                      # From DR intro tips
                       # <link id='2' value='Ever wondered about the time you've spent in Elanthia?  Check the PLAYED verb!' cmd='played' echo='played' />
-                      while data = $_SERVERSTRING_.match(/'([^=]*'[^=]*)'/)
+                      # From GS
+                      # <d cmd='forage Imaera's Lace'>Imaera's Lace</d>, <d cmd='forage stalk burdock'>stalk of burdock</d>
+                      while data = $_SERVERSTRING_.match(/'([^=>]*'[^=>]*)'/)
```

Comment:
The adjustment with double-quotes should probably be updated as well.

Reply:
Thanks for commenting, and making suggestions, btw.

I had intentionally left this line alone, since I haven't see the problem occur with the single quotes before. In theory it breaks nothing to add it, but without a problem to address (for all the other cases, I have example lines that get fixed, either from DR, or from GS) I rather not add the change. 

All changes so far I have tested robustly, across 9 DR characters, and GS testers have verified the others. Which is to say... with a good level of testing/exposure to far, we haven't run into this specific problem with single quotes.